sovtokenfees/sovtokenfees/test/taa/fees/test_no_taa_fees.py

```diff
@@ -0,0 +1,34 @@
+from sovtokenfees.constants import FEES
+
+from plenum.common.constants import NYM
+
+
+def test_nym_with_fees_with_taa(helpers,
```

Comment:
This is maybe not in master yet, but we will reject txns with TAA set if TAA is not set/enabled. So, these test will fail then.